coralproject/talk
# Story 139595043 Question Box to Comment Stream

## What does this PR do?

- Adds the option of save a question box to the settings of the comment stream
- The question box has an enable field AND a content field.
- It adds the question box to the top of the comment stream if it is enable in the settings for the stream.

## How do I test this PR?

- Get an admin user
- Enable the question box in the settings for the comment stream.
- Add the question 
- Go check that the question is at the top of the comment stream.


Story: https://www.pivotaltracker.com/story/show/139595043
